### ISSUE TITLE
Export mince function

### ISSUE THREAD

**Issue by dpsanders** [JuliaIntervals/IntervalArithmetic.jl#274]
```jl
function mince(X, n)
    r = range(X.lo, stop=X.hi, length=n+1)   # stop is required for Julia 1.0
    return [interval(r[i], r[i+1]) or i in 1:length(r)-1]
end
```

**Comment by lbenet** [comment]
It would be great to add it !!!

**Comment by lbenet** [comment]
Closed by #354
